### DISCUSSION TITLE
Using Scrobble to determine if a show or movie has been watched

### DISCUSSION THREAD

**Discussion by JB184351** [trakt/trakt-api#434] [Q&A] [answered]
My question is does the Trakt API have the ability to tell when a user has watched a show from a streaming service like Netflix for example? I see the Start, Progress, and Pause endpoints but I don't understand how they are used/work.

**Comment by tysonkerridge** [comment, marked as answer]
Assuming you're wanting to see on say the history item which service/app was the one that scrobbled, I don't think it currently keeps track of that, or at least doesn't surface it. For something like Netflix, they would need to have an official scrobbling tool plugged into Trakt so unless that happened you'd probably at best only get a reference to the third party app that applied the scrobble.

A scrobble is more or less just an automated way of adding things to history when its 'finished' including keeping progress when pausing, similar to how a streaming service or youtube keeps track of where you're at if you stop half way and then can resume. It won't actually "complete" until reaching a certain threshold or the end of the media.

**Reply by JB184351** [reply]
Makes sense and kinda of what I figured just had to ask for an idea for an app I wanted to make.
